Loan.ts

```diff
@@ -1378,5 +1378,15 @@ export default class Loan {
     response = await this.fineract_obj.put(path, loanCollateral_update_config);
     return response.data;
   }
+  async fetch_loan_details_byloanId(loanId: number): Promise<[any]> {
+
+    let search_query_obj = {
```

Comment:
search_query_obj should be passed as argument to the fetch_loan_details_byloanId